interpreter/llvm/src/include/llvm/IR/DebugInfoMetadata.h

```diff
@@ -541,7 +541,9 @@ class DIFile : public DIScope {
     return StringRefChecksum;
   }
   Optional<StringRef> getSource() const {
-    return Source ? Optional<StringRef>((*Source)->getString()) : None;
+    if (!Source || !*Source)
+      return None;
+    return Optional<StringRef>((*Source)->getString());
```

Comment:
Can you submit that change for review in llvm?

Reply:
Hi @vgvassilev. Sorry for the late reply. I'm not very keen to submit this to llvm, because each time I submitted something it either got completely ignored or someone did a separate equivalent commit instead of merging what I submitted.

Reply:
I changed my mind, I will give this a chance. I hope that it won't be ignored. This line is still the same in the latest llvm, I just checked.

Reply:
The change has been submitted here: https://reviews.llvm.org/D137152